x-pack/legacy/plugins/apm/public/components/app/Settings/AddSettings/AddSettingFlyoutBody.tsx

```diff
@@ -143,7 +169,7 @@ export function AddSettingFlyoutBody({ onSubmit }: { onSubmit: () => void }) {
           <EuiFieldNumber
             min={0}
             max={1}
-            step={0.1}
+            step={0.001}
```

Comment:
I think this might need to change to `0.01` as I don't think we need to support 0.5%. @graphaelli thoughts?

Reply:
Notes from Slack conversation: we agreed to keep the 0.001 validation because it should be possible for users to do 0.1% sample rate configuration.